text/0000-fabric-private-chaincode-1.0.md

```diff
@@ -381,7 +392,10 @@ However, any FPC Chaincode invocation will return an error because the FPC Chain
 
 The administrator of the peer hosting the FPC Chaincode Enclave is responsible for the initialization and registration of the enclave.
 The operation is performed by executing the `initEnclave` admin command (see Step 2 above).
-The command can be triggered through the FPC Client SDK (see [earlier Deployment Section](#deployment)).
+
+FPC implements this command as a regular chaincode method, and uses Fabric's `peer chaincode` command to call it.
+Hence, its implementation does not modify, nor requires any modifications, to the Fabric framework.
+Also, conveniently, the command can be triggered through the FPC Client SDK (see [earlier Deployment Section](#deployment)), which is in turn built on the Fabric Client SDK for Go.
```

Comment:
we could mention also the peer cli wrapper, also motivated by convenience (which it is really what it is for) and/or reference again to `fpc-management.api` where both are documented.